README.md

```diff
@@ -253,49 +258,98 @@ compensation.
+    vast number of potential shareholders.
+    Increasing the number of shareholders (or partial owners) of a company decreases that
+    company’s risk overall.
+    There are other reasons as well.
+  - We’ll discuss more details on IPO and
+    [what this all means for your equity](#what-is-equity-worth?) later on!
+
+🚧 Josh, we can move this material to the defining occurrence of IPO (now lives in What is
+Equity worth?)
+or more that defining occurrence up here.
+Tough to explain before defining stock, but Stock and Companies sections could easily be
+switched with no harm done.
+
+### What is stock?
+
+- 🄳 [**Stock**](https://en.wikipedia.org/wiki/Stock) is a legal invention that allows the company
+  to grant ownership to a variety of people or other companies in flexible ways.
   Stock is divided into **shares**, and every owner, or **shareholder**, holds a specific number
   of shares. Founders, investors, employees, board members, contractors, advisors, and other
   companies, like law firms, can all be shareholders.
 - 🄳 Stock ownership is formalized on **stock certificates**, which are fancy pieces of paper
   that prove you own the stock.
   - Sometimes you have stock but don’t have the physical certificate, as it may be held for
-    you at a law office.
+    you at a law office or by your stock broker.
   - 🄳 Some companies now manage their ownership through online services called **ownership
     management platforms**, such as [Carta](https://carta.com/). If the company you work for uses
     an ownership management platform, you will be able to view your stock certificates and
     stock values online.
-  - 🚧 Mention that stock may be taxable up front (and reference section).
 - 🄳 The [**outstanding shares**](http://www.investopedia.com/terms/o/outstandingshares.asp) are
   the total number of shares at a given point, reflecting how many shares are currently held
   by all shareholders.
   This number starts at an essentially arbitrary value (such as 10 million) when the company
   is created, and thereafter will increase as new shares are added (issued) and granted to
   people in exchange for money or services.
-  Outstanding shares may increase or decrease for other reasons too (such as stock splits
-  and share buybacks, which we won’t get into here).
-  There is a lot of nuance and complexity in all this, because each part is defined by piles
-  and piles of legal documents.
-  - 🚧 What is a good overview on stock splits and share buyback.
-    Key resources?
+  Outstanding shares may increase or decrease for other reasons too.
+  - In a [**stock split**](https://www.investopedia.com/terms/s/stocksplit.asp), for example, a
+    company’s board of directors decides to increase the number of outstanding shares by
+    doubling or tripling the number of shares each stockholder has.
+    Stock splits are a way to increase the liquidity of a company’s stock when the stock price
```

Comment:
This doesn't seem clear enough. I think we may need to reference the details of why this is might be helpful. At face value, it's not clear how a stock can be "too high for average buyers."
https://www.investopedia.com/terms/s/stocksplit.asp
https://www.investopedia.com/articles/01/072501.asp
Also reverse stock splits not mentioned.

Reply:
(As in, most readers would wonder, "can't you just buy fewer shares if the price is high?")

Reply:
Hm. Let's add that "can't you just buy fewer shares" to questions for Eric.

Reply:
I think we have the info and can write and then get it reviewed. The actual reasons are covered in those Investopedia links—it has to do with board lots, bid-ask spread, and psychological factors. I don't think we need it covered in detail though, but just summarized precisely.

Reply:
Basically, the at face value the concept makes little sense, but it's all these more subtle considerations.